python/dgl/distributed/graph_partition_book.py

```diff
@@ -750,7 +802,8 @@ def shared_memory(self, graph_name):
 
         eid_range = [None] * len(self.etypes)
         for i, etype in enumerate(self.etypes):
-            eid_range[i] = (etype, self._typed_eid_range[etype])
+            c_etype = self.to_canonical_etype(etype)
+            eid_range[i] = (c_etype, self._typed_eid_range[c_etype])
```

Comment:
the key of `_typed_eid_range` is canonical etype?

Reply:
if inited by new partition config(will be enabled in the next PR), it's canonical. If inited by old partition config, it's non-canonical.